### DISCUSSION TITLE
Keycloak 23.0.4 CORS configuration

### DISCUSSION THREAD

**Discussion by VincenzoRotondi** [keycloak/keycloak#26331] [Q&A] [answered]
I'm trying to log in via API from an Angular client on Keycloak on a different server but it keeps giving me the error "has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource ."

Searching online they say to set "+" to Web origin in Realm settings, but in my version that field is not there. The closest thing I've found is Valid request URIs in clients>admin-cli>client details>advanced, but setting "+" in that field doesn't resolve the error.

What can I do? Thank you

**Comment by VincenzoRotondi** [comment, marked as answer]
Ok, I figured out how to do it.
You had to enable Standard flow in clients>admin-cli>client details>settings to make the Web Origins field appear and then set *

**Comment by borczeangelovbasenetch** [comment]
Similar issue in Version 25.0.2. Link: https://github.com/keycloak/keycloak/issues/32489
